ledger-tool/src/args.rs

```diff
@@ -53,6 +53,13 @@ pub fn accounts_db_args<'a, 'b>() -> Box<[Arg<'a, 'b>]> {
             .validator(is_pow2)
             .takes_value(true)
             .help("Number of bins to divide the accounts index into"),
+        Arg::with_name("accounts_index_initial_accounts")
```

Comment:
maybe this should indicate that we aim to prepare capacity of the accounts index, so `accounts_index_initial_capacity` - a plus is that this implies the count of entries, otherwise it would be nice to indicate that this flag specifies a number of accounts (e.g. `_initial_accounts_count`), since now it may look like you are listing some hardcoded accounts to initialize